src/resources/Secrets/SecretDetails.js

```diff
@@ -43,9 +41,34 @@ export function SecretDetails(props) {
     },
   ];
 
+  const Events = () => (
+    <EventsList
+      key="events"
+      namespace={props.namespace}
+      filter={filterByResource('Secret', props.resourceName)}
+      hideInvolvedObjects={true}
+    />
+  );
+
+  const Configuration = secret => (
+    <UI5Panel
+      fixed
+      keyComponent={'storageclass-configuration'}
+      title={t('storage-classes.headers.configuration')}
```

Comment:
move this to `common.headers` instead of using storage-class and also adjust the resort of the places in app where storage-class translation is used